tensorflow/contrib/rnn/kernels/BUILD

```diff
@@ -0,0 +1,19 @@
+# Libraries and Kernels for RNNs.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+load("//tensorflow:tensorflow.bzl", "tf_kernel_library")
+
+tf_kernel_library(
+    name = "lstm_ops",
+    prefix = "lstm",
+    deps = [
+        "//tensorflow/contrib/rnn:lstm_ops_op_lib",
+        "//tensorflow/core:framework",
```

Comment:
Unfortunately, `tf_gen_op_libs` adds `core:framework` to deps. I replaced the call to `tf_gen_op_libs` with this :

    cc_library(
      name = "lstm_ops_op_lib",
      srcs = ["ops/lstm_ops.cc"],
      copts = tf_copts(),
      deps=(["//tensorflow/core:framework_headers_lib", "//google/protobuf"]),
      alwayslink=1,
      linkstatic=1,
     )

Also, `tf_kernel_library` brings in `core:framework` too :( You have to change that to a `cc_library`. Sorry about the issues. Things are moving fast, and I have to run as fast as I can even to stay in the same place.